src/routes/disclaimer/DisclaimerContainer.js

```diff
@@ -59,7 +60,10 @@ class DisclaimerContainer extends React.Component<DisclaimerPropsType, Disclaime
     setTimeout(() => this.setState({ timeoutExpired: true }), LOADING_TIMEOUT)
 
     try {
-      const disclaimerEndpoint = createDisclaimerEndpoint(baseUrl)
+      const appSettings = new AppSettings()
+      const apiUrlOverride = await appSettings.loadApiUrlOverride()
+
+      const disclaimerEndpoint = createDisclaimerEndpoint(apiUrlOverride || baseUrl)
```

Comment:
Is there already a test for DisclaimerContainer? If not then this is fine.

Reply:
There is no test yet.